packages/models-library/src/models_library/api_schemas_directorv2/clusters.py

```diff
@@ -156,13 +143,27 @@ class Config(BaseCluster.Config):
                         "password": "somepassword",
                     },
                     "accessRights": {
-                        154: CLUSTER_ADMIN_RIGHTS,
-                        12: CLUSTER_MANAGER_RIGHTS,
-                        7899: CLUSTER_USER_RIGHTS,
+                        154: CLUSTER_ADMIN_RIGHTS,  # type: ignore[dict-item]
+                        12: CLUSTER_MANAGER_RIGHTS,  # type: ignore[dict-item]
+                        7899: CLUSTER_USER_RIGHTS,  # type: ignore[dict-item]
                     },
                 },
             ]
         }
+    )
+
+    @field_validator("thumbnail", mode="before")
```

Comment:
the always=True disappeared. is this not necessary anymore?

Reply:
I added `validate_default=True` in the related `thumbnail` field.

Reply:
but the field did not really disappear it seems. what is the differnece?